quattor/client/rpms.pan

```diff
@@ -5,21 +5,32 @@
 
 unique template quattor/client/rpms;
 
+@{
+descr = variable to enable installation of ncm-cdispd and ccm-listend
+values = boolean
+default = true
+required = No
+}
+variable QUATTOR_INSTALL_CDISPD ?= true;
+
 include 'quattor/client/version';
 
 '/software/packages' = {
     # python-elementtree is required by YUM on SL5 but not listed as a dependency
     # of any other package
     if ( OS_VERSION_PARAMS['majorversion'] == '5' ) {
-      pkg_repl('python-elementtree');
+        pkg_repl('python-elementtree');
```

Comment:
While this might be useful clean up, it's confusing to find a line touched by git blame that then has no explanation in the commit log. Perhaps just leave this whitespace untouched until we have a reason to touch this line? (Or do a one-off global whitespace clean up in the entire file?)